src/main/java/com/amplitude/Amplitude.java

```diff
@@ -2,6 +2,7 @@
 
 import java.util.*;
 import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.Future;
```

Comment:
Why do we need to import this?

Reply:
Good catch, let me remove it. This was added when I play around the suggestion to return a Future and later we picked another way.